src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1110,10 +1110,32 @@ private async Task SendInviteAsync(OrganizationUser orgUser, Organization organi
                 throw new BadRequestException("Invalid token.");
             }
 
+            ICollection<Policy> orgPolicies = null;
+            var userOrgs = await _organizationUserRepository.GetManyByUserAsync(user.Id);
+            if (userOrgs.Count > 0)
+            {   
+                orgPolicies = await _policyRepository.GetManyByOrganizationIdAsync(orgUser.OrganizationId);
+                if (orgPolicies.Any(policy => policy.Type == PolicyType.OnlyOrg && policy.Enabled))
+                {
+                    throw new BadRequestException("You cannot join this organization until you are not a " +
+                        "part of any other organizations.");
```

Comment:
You cannot join this organization while you are a member of another organization.

Reply:
Done